### ISSUE TITLE
Please provide tracking-free version

### ISSUE THREAD

**Issue by IzzySoft** [CreaRo/DAWebmail#52]
With the latest version, you introduced *Firebase Analytics.* Some folks decided to use Zimbra as an alternative to being tracked by Google when using GMail – so it would be unfortunate to bring that tracking back.

Could you at least have a Firebase-free `.apk` attached to `releases/`? And by "free" I do not simply mean "disabled", but "removed" (i.e. "contains no Firebase code"). Thanks in advance!

**Comment by crearo** [comment]
Alrighty! I'm willing to do so. Can you please submit a PR that removes Firebase from the source code completely?

**Comment by IzzySoft** [comment]
If I could, I'd done that instead of opening an issue :wink: Sorry, but I'm no Android dev.

**Comment by crearo** [comment]
Briefly went through your github profile - doesn't look like DAIICT is your alma mater - the university this app was originally made for. 

- Are you using this app for Zimbra's webmail service elsewhere? I'd love to list it down on the readme if that's the case.
- or, do you have a tracker that looks at github repos and checks to see if anything Google, ie, Firebase in this case, is added to well-off open source repos. Asking because you seem to be linked to FDroid.

**Comment by IzzySoft** [comment]
No, I'm not affiliated either with the map nor with DAIICT. But yes, I'm affiliated with F-Droid and also run my own F-Droid repo.

* I'm not using Zimbra nor its web services (as for DAIICT: from the readme it didn't sound like your app is bound to that – but rather could be used with any Zimbra Email Service?)
* I have a personal tracker (me, skimming manually for interesting open-source Android apps from time to time) – which is how I found your project. Checking the `releases/` page I saw your announcement "Added firebase analytics".

Firebase is a show-stopper at F-Droid. And I personally would not want to use an app having that integrated – especially not if the app deals with sensitive content (like mails do from time to time). As I know others feel the same, I thought to bring it up. I'm a "privacy advocate", so I try to help the privacy-aware community :wink:

**Comment by crearo** [comment]
Great. Yes, it isn't specific to DA-IICT, just any org using Zimbra. 

I agree with you, and I'm not really doing much with the analytics as it is. I'll remove all Firebase code from the repo this coming weekend :) Will close this issue once this is done.

**Comment by IzzySoft** [comment]
Thanks a lot! I'm already providing your app [via my repo](https://apt.izzysoft.de/fdroid/index/apk/com.sigmobile.dawebmail). Once the Firebase-free version was fetched, I'll remove the `Tracking` AntiFeature. Guess you'll want to keep GMS for cloud messages (which makes it not fitting on F-Droid's main repo, but is OK for mine). If you consider replacing that as well (to make your app fully FOSS) but still want those messages, there are ways to achieve this (e.g. [Tutanota has implemented a FOSS alternative](https://f-droid.org/en/2018/09/03/replacing-gcm-in-tutanota.html) called SSE (Server Sent Events)).

**Comment by ghost** [comment]
Any news here ... ? Would love to give it a try

**Comment by crearo** [comment]
Sure, I don't think I ever got the time to do this. If you're willing, go ahead and remove Firebase. If you don't have the time, I'll make sure this happens this month ;)

**Comment by IzzySoft** [comment]
I'm not a dev, so I cannot do it. So I'm patiently looking forward to your action, @crearo 😇 Thanks!

**Comment by crearo** [comment]
Ah thanks @IzzySoft. this was actually in response to @nr458h though.

**Comment by IzzySoft** [comment]
Ugh… :scream: Apologies :smile:
